etc/config/compiler-explorer.defaults.properties

```diff
@@ -13,6 +13,8 @@ allowedShortUrlHostRe=^([-a-z.]+\.)?(xania|godbolt)\.org$
 googleShortLinkRewrite=^https?://goo.gl/(.*)$|https://godbolt.org/g/$1
 
 wine=/opt/wine-devel/bin/wine64
+python3=/usr/local/bin/python3
```

Comment:
In the amazon instances, note that:
 ```
~$ which python3
/usr/bin/python3
```

Reply:
I think we need to add /usr/bin then to export ALLOWED_FOR_READ in /c-preload/compile-wrapper? But maybe it would be better if we installed python to /opt/?

Reply:
I will use /usr/bin/python3 for now.